packages/altair-static/src/index.ts

```diff
@@ -92,6 +92,14 @@ export interface RenderOptions {
      * }
      */
     initialSettings?: {[key: string]: any};
+
+    /**
+     * Initial subscription provider
+     *
+     * @default "websocket"
+     */
+    initialSubscriptionProvider?:
```

Comment:
I'll need to take a look at this again. Either we somehow import the types from `altair-app` or we type this to `string`. If we hardcode the string constants here, we can easily miss updating this when a new subscription provider is added.

Reply:
yeah, this specific one I just put the strings since altair-static doesn't have any dependency from where to import it